opensfm/src/bundle/bundle_adjuster.h

```diff
@@ -128,6 +130,23 @@ struct BAFisheyeCamera : public BACamera{
   void SetK2(double v) { parameters[BA_CAMERA_K2] = v; }
 };
 
+struct BADualCamera : public BACamera{
+  double parameters[BA_CAMERA_NUM_PARAMS];
```

Comment:
Should we add a new enum with `BA_DUAL_CAMERA_NUM_PARAMS` so that `perspective` and `fisheye` cameras keep their 3 parameters instead of getting 4?